src/views/HomeView.vue

```diff
@@ -1,26 +1,12 @@
-<<<<<<< HEAD
 <script setup>
 import Typograpy from '../components/Typography.vue'
 import ColorsIconography from '../components/ColorsIconography.vue'
 
-=======
-<script setup type="js">
-import InputBox from '../components/InputBoxes.vue'
-import Buttons from '../components/Buttons.vue'
-import Tables from '../components/Tables.vue'
->>>>>>> 42b905e4b00b13ceefaea2d98320c011e10e7060
 </script>
 
 <template>
   <main>
-<<<<<<< HEAD
-    <Typograpy />
-    <!-- <ColorsIconography /> -->
-=======
-    <InputBox />
-    <!-- <Buttons /> -->
-    <Tables />
-  
->>>>>>> 42b905e4b00b13ceefaea2d98320c011e10e7060
```

Comment:
eto rin dinelete mo ata yung existing na code n ginawa ni melanio paki coordinate to sa kanya